ff-uint_derive/src/lib.rs

```diff
@@ -882,20 +800,20 @@ fn prime_field_impl(
 
             #[inline]
             fn wrapping_add(self, other: #name) -> Self {
-                #name(self.0 + other.0).reduced()
+                #name(self.0.wrapping_add(other.0)).reduced()
```

Comment:
@voidxnull at lib.rs ops should be unchecked here and below, because we cannot reach overflow on valid Field elements.